controllers/volumereplication_controller.go

```diff
@@ -84,9 +85,13 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return reconcile.Result{}, err
 	}
 
+	instance.Status.LastStartTime = metav1.Now()
+	r.updateStatus(instance, replicationv1alpha1.Replicating, "")
```

Comment:
This should be set only in case of success?

Reply:
Done